message.go

```diff
@@ -96,7 +103,33 @@ func sendDiscordMessage(s *discordgo.Session, channelID string, msg Message) err
 		embedFull.Color = 0x000000
 	}
 
-	_, err := s.ChannelMessageSendEmbed(channelID, embedFull)
-	return err
+	message := &discordgo.MessageSend{
+		Embeds: []*discordgo.MessageEmbed{
+				embedFull,
+		},
+		Components: []discordgo.MessageComponent{
+			discordgo.ActionsRow{
+				Components: []discordgo.MessageComponent{
+					discordgo.Button{
+						Label: "Delete Message",
+						Style: discordgo.DangerButton,
+						Disabled: false,
+						CustomID: "response_delete",
+					},
+				},
+			},
+		},
+	}
 
-}
+	sentMsg, err := s.ChannelMessageSendComplex(channelID, message)
+
+	s.AddHandler(func(s *discordgo.Session, i *discordgo.InteractionCreate) {
+        if i.Type == discordgo.InteractionMessageComponent && i.MessageComponentData().CustomID == "response_delete" {
+            err := s.ChannelMessageDelete(channelID, sentMsg.ID)
+			if err != nil {
+			fmt.Println("Error occurred during deletion:" , err)
```

Comment:
Missing a tab here, should be tabbed over one

Reply:
resolved